File: module06/module025/module2/src/main/java/com/android/example/module06_module025_module2/ClassAAI.java
package com.android.example.module06_module025_module2;

public class ClassAAI {
	private 	io.reactivex.internal.schedulers.IoScheduler instance_var_1_0 = new io.reactivex.internal.schedulers.IoScheduler();
	private 	io.reactivex.subscribers.TestSubscriber<Object> instance_var_1_1 = new io.reactivex.subscribers.TestSubscriber();

	public void method0(
		io.reactivex.subscribers.TestSubscriber<Object> param0) throws Throwable {
	}

	public void method1(
		io.reactivex.subscribers.TestSubscriber<Object> param0) throws Throwable {
		for (int iAb = 0; iAb < 5; iAb++) {
			java.util.Collections.emptyList().forEach( lambda0 -> {
				try {
					for (int iAc = 0; iAc < 0; iAc++) {
						param0.onError(new java.lang.Throwable());

						io.reactivex.subscribers.TestSubscriber<Object> local_var_5_0 = new io.reactivex.subscribers.TestSubscriber();
						local_var_5_0.isCancelled();

						io.reactivex.subscribers.TestSubscriber<Object> local_var_5_1 = new io.reactivex.subscribers.TestSubscriber();
						local_var_5_1.assertSubscribed();

						io.reactivex.internal.schedulers.SingleScheduler local_var_5_2 = new io.reactivex.internal.schedulers.SingleScheduler();
						local_var_5_2.createWorker();

					}
				} catch(Throwable e) { } // ignore
			});
		}
		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_1 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_1.assertNotSubscribed();

		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_2 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_2.isCancelled();

	}

	public void method2(
		io.reactivex.subscribers.TestSubscriber<Object> param0,
		io.reactivex.subscribers.TestSubscriber<Object> param1,
		io.reactivex.internal.schedulers.SingleScheduler param2) throws Throwable {
		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_3 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_3.isDisposed();

		io.reactivex.observers.TestObserver<Object> local_var_2_4 = new io.reactivex.observers.TestObserver();
		local_var_2_4.isDisposed();

		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_5 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_5.hasSubscription();

	}

	public void method3(
		io.reactivex.subscribers.TestSubscriber<Object> param0,
		io.reactivex.subscribers.TestSubscriber<Object> param1,
		io.reactivex.observers.TestObserver<Object> param2,
		io.reactivex.subscribers.TestSubscriber<Object> param3) throws Throwable {
		io.reactivex.internal.disposables.ListCompositeDisposable local_var_2_4 = new io.reactivex.internal.disposables.ListCompositeDisposable();
		local_var_2_4.isDisposed();

		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_5 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_5.assertNotSubscribed();

	}
}

File: module06/module293/module3/src/main/java/com/android/example/module06_module293_module3/ClassAAI.kt
package com.android.example.module06_module293_module3

@Suppress("UNUSED_PARAMETER")
class ClassAAI {
	private 	val instance_var_1_0: io.reactivex.internal.util.LinkedArrayList = io.reactivex.internal.util.LinkedArrayList(76)
	private 	val instance_var_1_1: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
	private 	val instance_var_1_2: io.reactivex.internal.disposables.ListCompositeDisposable = io.reactivex.internal.disposables.ListCompositeDisposable()
	private 	val instance_var_1_3: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()

	fun method0(
		param0: io.reactivex.subscribers.TestSubscriber<kotlin.Any>
	) {
		if (kotlin.Any().equals(kotlin.Any())) {
			if (kotlin.Any().equals(kotlin.Any())) {
				val local_var_4_0: io.reactivex.internal.schedulers.ComputationScheduler = io.reactivex.internal.schedulers.ComputationScheduler()
				local_var_4_0.createWorker()

			}
		}
		val local_var_2_1: io.reactivex.internal.util.LinkedArrayList = io.reactivex.internal.util.LinkedArrayList(31)
		local_var_2_1.toString()

		val local_var_2_2: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_2.assertNotSubscribed()

	}

	fun method1(
		param0: io.reactivex.internal.schedulers.IoScheduler,
		param1: io.reactivex.internal.util.LinkedArrayList,
		param2: io.reactivex.subscribers.TestSubscriber<kotlin.Any>
	) {
		param0.createWorker()

		val local_var_2_3: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_3.assertNotSubscribed()

		val local_var_2_4: io.reactivex.internal.disposables.ListCompositeDisposable = io.reactivex.internal.disposables.ListCompositeDisposable()
		local_var_2_4.dispose()

		param0.start()

	}

	fun method2(
		param0: io.reactivex.subscribers.TestSubscriber<kotlin.Any>,
		param1: io.reactivex.subscribers.TestSubscriber<kotlin.Any>,
		param2: io.reactivex.subscribers.TestSubscriber<kotlin.Any>
	) {
		param0.onError(kotlin.Throwable())

		param0.onError(kotlin.Throwable())

		val local_var_2_3: io.reactivex.disposables.SerialDisposable = io.reactivex.disposables.SerialDisposable()
		local_var_2_3.dispose()

		param0.isDisposed()

	}
}
